src/swaps/fastbtc/FastbtcSwapProvider.js

```diff
@@ -0,0 +1,254 @@
+import { io } from 'socket.io-client'
+import BN from 'bignumber.js'
+import { mapValues } from 'lodash-es'
+import { SwapProvider } from '../SwapProvider'
+import { v4 as uuidv4 } from 'uuid'
+import { chains, currencyToUnit, unitToCurrency } from '@liquality/cryptoassets'
+import { withInterval } from '../../store/actions/performNextAction/utils'
+import { prettyBalance } from '../../utils/coinFormatter'
+import cryptoassets from '@/utils/cryptoassets'
+import { getTxFee } from '../../utils/fees'
+
+const fastBtcSatoshiFee = 5000
+const fastBtcPercentageFee = 0.2
+
+class FastbtcSwapProvider extends SwapProvider {
+  constructor (config) {
+    super(config)
+    this.socketConnection = io(this.config.agent, {
+      reconnectionDelayMax: 10000
+    })
+
+    this.socketConnection.on('connect', function () {
+      console.log('FatBtc socket connected')
+    })
+
+    this.socketConnection.on('disconnect', function () {
+      console.log('FastBtc socket disconnected')
+    })
+  }
+
+  async getSupportedPairs () {
+    return []
```

Comment:
Follow what `LiqualitySwapProvider` does for this one
`[]` should be returned when markets are unbounded. E.g. in AMMs like uniswap. In this case it's only BTC -> RBTC